src/components/bulk-email-tool/bulk-email-task-manager/bulk-email-scheduled-emails-table/BulkEmailScheduledEmailsTable.jsx

```diff
@@ -96,7 +97,7 @@ function BulkEmailScheduledEmailsTable({ intl }) {
       },
     } = row;
     const dateTime = new Date(taskDueUTC);
-    const emailRecipients = targets.replaceAll('-', ':').split(', ');
+    const emailRecipients = targets.replaceAll('-', ':').split(', ').map(getRecipientFromDisplayText);
```

Comment:
Reversing the "display text" like this is not a very robust solution. Can you just modify `flattenScheduledEmailsArray` so that it stores _both_ the `email.courseEmail.targets` IDs and the display text on `row.original`  ? Then you won't have to do this reversing.